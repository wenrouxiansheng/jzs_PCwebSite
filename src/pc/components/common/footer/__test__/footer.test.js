import React from "react";
import renderer from "react-test-renderer";
import Footer from "../index";
import { BrowserRouter as Router } from "react-router-dom";

describe("PC:<Footer />", () => {
  it("render", () => {
    const wrapper = renderer.create(
      <Router>
        <Footer />
      </Router>
    );
    let tree = wrapper.toJSON();

    //渲染测试
    expect(tree).toMatchSnapshot();
  });
});
