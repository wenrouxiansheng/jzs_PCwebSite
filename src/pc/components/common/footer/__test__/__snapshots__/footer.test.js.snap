// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PC:<Footer /> render 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="nav"
    >
      <dl>
        <dt>
          关于我们
        </dt>
        <dd>
          <a
            className=""
            href="/site/pc/aboutUs/teachers"
            onClick={[Function]}
          >
            师资力量
          </a>
        </dd>
        <dd>
          <a
            className=""
            href="/site/pc/brandIntroduction"
            onClick={[Function]}
          >
            教学模式
          </a>
        </dd>
        <dd>
          <a
            className=""
            href="/site/pc/brandIntroduction/course"
            onClick={[Function]}
          >
            课程研发
          </a>
        </dd>
        <dd>
          <a
            className=""
            href="/site/pc/aboutUs/contactUs"
            onClick={[Function]}
          >
            加入我们
          </a>
        </dd>
        <dd>
          <a
            className=""
            href="/site/pc/brandIntroduction"
            onClick={[Function]}
          >
            桔子树介绍
          </a>
        </dd>
      </dl>
      <dl>
        <dt>
          课程设置
        </dt>
        <dd>
          <a
            className=""
            href="/site/pc/vocalMusic"
            onClick={[Function]}
          >
            声乐培训
          </a>
        </dd>
        <dd>
          <a
            className=""
            href="/site/pc/danceTraining"
            onClick={[Function]}
          >
            舞蹈培训
          </a>
        </dd>
        <dd>
          <a
            className=""
            href="/site/pc/instrumentalMusic"
            onClick={[Function]}
          >
            器乐培训
          </a>
        </dd>
        <dd>
          <a
            className=""
            href="/site/pc/fineArt"
            onClick={[Function]}
          >
            美术培训
          </a>
        </dd>
      </dl>
      <dl>
        <dt>
          精彩视频
        </dt>
        <dd>
          <a
            className=""
            href="/site/pc/aboutUs/studentHonor"
            onClick={[Function]}
          >
            歌唱大赛
          </a>
        </dd>
        <dd>
          <a
            className=""
            href="/site/pc/aboutUs/studentHonor"
            onClick={[Function]}
          >
            才艺大赛
          </a>
        </dd>
        <dd>
          <a
            className="btnsh_udesk_im"
            href="/"
            onClick={[Function]}
          >
            教学视频
          </a>
        </dd>
      </dl>
      <dl>
        <dt>
          服务专区
        </dt>
        <dd>
          <a
            className="btnsh_udesk_im"
            href="/"
            onClick={[Function]}
          >
            我要报名
          </a>
        </dd>
        <dd>
          <a
            className="btnsh_udesk_im"
            href="/"
            onClick={[Function]}
          >
            免费试听
          </a>
        </dd>
        <dd>
          <a
            className=""
            href="/site/pc/campusDistribution"
            onClick={[Function]}
          >
            校区地址
          </a>
        </dd>
        <dd>
          <a
            className=""
            href="/site/pc/aboutUs/contactUs"
            onClick={[Function]}
          >
            约课电话
          </a>
        </dd>
      </dl>
    </div>
    <div
      className="infoBox"
    >
      <div
        className="QRBox"
      >
        <div>
          <img
            alt=""
            src="test-file-stub"
          />
          <p>
            扫码关注公众号
          </p>
        </div>
        <div>
          <img
            alt=""
            src="test-file-stub"
          />
          <p>
            扫码下载app
          </p>
        </div>
      </div>
      <p>
        <img
          alt=""
          src="test-file-stub"
        />
        联系我们:
        <strong>
          400-900-8898
        </strong>
      </p>
      <p>
        <img
          alt=""
          src="test-file-stub"
        />
        北京市通州区新华西街60号万达广场A座30层
      </p>
    </div>
  </div>
  <p>
    商务合作：010-5965-2885
  </p>
  <p>
    京ICP备17024125号-1
  </p>
  <p>
    版权所有@2021.北京桔子树文化传播有限责任公司
  </p>
  <p>
    <a
      href="http://www.beian.gov.cn/portal/registerSystemInfo?recordcode=11011202001383"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt=""
        src="test-file-stub"
      />
       京公网安备 11011202001383号
    </a>
  </p>
</footer>
`;
